### ISSUE TITLE
Game Crashes when doing OBS Game Capture 

### ISSUE THREAD

**Issue by JasonBean-99** [juliand665/Dynamic-FPS#225]
So when I loaded up OBS and added a game capture, I went to select minecraft but after I waited a bit, the game just simply closed. I went into the config and disabled the mod, and the capture remained and worked perfectly like normal. 

Those were the logs from when the game closed. 
[2024-09-23-6.log](https://github.com/user-attachments/files/17101356/2024-09-23-6.log)

**Comment by LostLuma** [comment]
That log does not have any information about the game closing at all.
Do you have an actual crash report in the `crash-reports` directory?

**Comment by JasonBean-99** [comment]
Weird, I had no crash logs at all, I look everywhere for one.

**Comment by LostLuma** [comment]
Does this also happen if you only have Dynamic FPS and no other mods?
If it does still happen could you see if it crashes with [`v3.0.1`](https://modrinth.com/mod/dynamic-fps/version/sejsYSOg) (Fabric / Quilt only)?

**Comment by JasonBean-99** [comment]
Yep just tried Fabric with 3,0,1 with only the mod, same thing happened

**Comment by JasonBean-99** [comment]
> Hmm, I think I am a little out of my depth with this issue then and do not know what you'd do to resolve it. You might want to see if you have OBS and (graphics) driver updates, and if those don't help ask in an OBS-specific space.

Yeah I figured it was just a conflict with OBS itself somehow,  I'll try leaving the issue in the OBS forums and see if I get anything, If I do I'll leave it here.  Thanks for checking though!
